src/main/java/com/hltech/store/versioning/MultipleVersionsBasedVersioning.java

```diff
@@ -3,7 +3,16 @@
 import java.util.HashMap;
 import java.util.Map;
 
-public class MultipleEventVersionPolicy<E> implements EventVersionPolicy<E> {
+/**
+ * In this strategy, multiple versions of the event has to bo supported in the application code.
+ * The application must contain knowledge of all deprecated event versions in order to support them.
+ * To avoid that consider using {@link UpcastingBasedVersioning}
+ *
+ * <p>Please note, that using this strategy is recommended only if you have one instance of you application running at the same time.
+ * Using this strategy in multi instance case, leads to the situation, where all instance must be updated
```

Comment:
Using this strategy in multi instance case leads to the situation, where all instances must be updated